distribution/docs/src/main/resources/content/_configuring/query-feedback.adoc

```diff
@@ -0,0 +1,70 @@
+:title: Configuring Query Feedback for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 04
+:summary: Configuring Query Feedback for ${catalog-ui}.
+
+== {title}
+
+${catalog-ui} provides an option to allow users to submit Query Feedback.
+
+.Configuring Query Feedback
+. First, configure the *Email Service* to point to a mail server. See <<org.codice.ddf.platform.email.impl.SmtpClientImpl, Email Service Configurations>>.
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-ui}* application.
+. Select the *Configuration* tab.
+. Select *Catalog UI Search*.
+. Select the *Enable Query Feedback* option to enable the query comments option for users in ${catalog-ui}.
+. Add a *Query Feedback Email Subject Template*.
+. Add a *Query Feedback Email Body Template*. The template may include HTML formatting.
+. Add the *Query Feedback Email Destination*.
```

Comment:
❓ The emails that get sent look exactly as configured without anything else, right? 
I was wondering if it provided any information about the query at all.

Reply:
Emails may include information about the query, as described in the `Query Feedback Template Replacements` section in the documentation
![image](https://user-images.githubusercontent.com/8041246/39155032-49cc26ec-4705-11e8-80d4-d8733bc89d02.png). Is this clear enough? Do you have any suggestions for improvements?

Reply:
I think this is fine for now, but it might be interesting to see a sample of what the output might look like. This could be a future enhancement. I think having this documentation is already a great improvement!